microsoftgraph/microsoft-graph-explorer
# Make GE respect single sign-out

## Overview

Graph explorer should respect single sign-out. If a user signs out of any Microsoft application, they should be logged out of GE if also logged in with the same account

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

To view the process locally, the only assumption is that you have installed a tunneling software e.g. ngrok. In your application portal, set the logout url to the one generated by ngrok. eg https://92832de0.ngrok.io/logout

## Testing Instructions

* Run Graph Explorer and log in
* Log in somewhere else :-) with the same account
* Logout of the other application
* Notice that you have been logged out of Graph Explorer
